server/game/core/card/LeaderCard.ts

```diff
@@ -4,14 +4,33 @@ import Contract from '../utils/Contract';
 import { CardType } from '../Constants';
 
 export class LeaderCard extends InPlayCard {
+    protected _deployed = false;
+
+    protected setupLeaderUnitSide;
+
+    public get deployed() {
+        return this._deployed;
+    }
+
     public constructor(owner: Player, cardData: any) {
         super(owner, cardData);
         Contract.assertEqual(this.printedType, CardType.Leader);
 
-        // TODO LEADER: add deploy epic action (see Base.ts for reference)
+        this.setupLeaderUnitSide = false;
+        this.setupLeaderSideAbilities();
     }
 
-    public override isLeader() {
+    public override isLeader(): this is LeaderCard {
         return true;
     }
-}
\ No newline at end of file
+
+    // this is overriden in the LeaderUnit derived class
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    public deploy() {}
```

Comment:
Do we need to have the deploy function here if only the Unit side cares about it? If so I believe we can label this function `public abstract deploy() : void` in order to avoid having to use the linter override

Reply:
Oh yeah, normally I would prefer that but `abstract` doesn't work with mixins unfortunately :/ and if I have to choose between the two I'd much rather have mixins